apps/wallet/src/ui/app/index.tsx

```diff
@@ -83,6 +84,11 @@ const App = () => {
 			</Route>
 
 			<Route path="welcome" element={<WelcomePage />} />
+			<Route
```

Comment:
Should we make this enabled just for beta or dev currently? Or feature flag it at least

Reply:
Oopsie - this was a bad rebase, I'll set up a feature flag to start integrating the new UI flows in a follow-up. I'd prefer that to enabling it for beta/dev mode since it would make dev quite difficult given the lack of functionality atm 😆